docs/contracts/accounting-oracle.md

```diff
@@ -21,3 +25,511 @@ This would ultimately result in no oracle reports and no stETH rebases for this
 :::
 
 [1]: /contracts/lido
+[2]: /contracts/lido-locator
+[3]: /contracts/legacy-oracle
+[4]: /contracts/burner
+[5]: /contracts/hash-consensus
+
+Access to lever methods is restricted using the functionality of the
+[AccessControlEnumerable](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/utils/access/AccessControlEnumerable.sol)
+contract and a bunch of [granular roles](#permissions).
+
+
+## Constants
```

Comment:
Constants, imho, worth to be shifted further and `ReportData` is brought up, as most important section.

Reply:
done